RELEASE/scripts/autoscend/autoscend_header.ash

```diff
@@ -434,7 +434,9 @@ boolean auto_harvestBatteries();
 int batteryPoints(item battery);
 int totalBatteryPoints();
 boolean batteryCombine(item battery);
-boolean auto_getBattery(item battery, int qty);
```

Comment:
header file was incorrect. it was `boolean auto_getBattery(item battery);`
not `boolean auto_getBattery(item battery, int qty);`